pom.xml

```diff
@@ -86,7 +86,7 @@
         <powerauth.version>0.24.0-SNAPSHOT</powerauth.version>
         <pushy.version>0.13.10</pushy.version>
         <swagger.version>2.9.2</swagger.version>
-        <unirest.version>1.4.9</unirest.version>
+        <unirest.version>3.4.01</unirest.version>
```

Comment:
Could we use higher version here, 3.4.04?
Or ideally the latest one https://mvnrepository.com/artifact/com.konghq/unirest-java

Reply:
Good point... latest is 3.6.01 - IMO it is typo in their docs here: http://kong.github.io/unirest-java/

Reply:
Fixed - I updated all libs...

Reply:
ok!

:) that's a classic case with the documentation and reality